src/components/common/navbar/Navbar.js

```diff
@@ -101,8 +101,10 @@ class Navbar extends Component {
 // component => wrapped component
 export default withCSS(Navbar, [
   'button',
+  'checkbox', // HACK need to import this in correct file
```

Comment:
@rschlaefli Could you please import this into ConfusionBarometer and FeedbackChannel? :)